warehouse/cli/hashing.py

```diff
@@ -28,6 +28,13 @@ def hashing():
 
 
 @hashing.command()
+@click.option(
+    "-e",
+    "--event-type",
+    type=click.Choice(["user", "project", "file", "organization", "team"]),
```

Comment:
question: how wild would it be to make this dynamic by seeing what inherits from HasEvents?

Reply:
it'd be wild, since we'd need to know about what models have events, and our CLI will not know about models until execution, so we don't import any code during the cli resolution itself.